### DISCUSSION TITLE
有关地理可视化相关的技术（方案），是会优先在 l7 实现吗

### DISCUSSION THREAD

**Discussion by Gao-pw** [antvis/G2#6351] [General]
最近在做地理可视化方面的一些技术调研，网上相关的文章 博客大多不是 `g2@5` 的版本，且 g2 文档在关于 **地理可视化** 层面的内容比较少（可能是我没找到），未来会将重心偏向 l7 吗？

**Comment by hustcc** [comment]
- 统计图表的用 G2
- 表格用 S2
- 图可视化相关的用 G6
- 地图可视化用 L7
- 流程编排之类的 X6

参考官网顶部的产品菜单

![image](https://github.com/antvis/G2/assets/7856674/c0c47c95-b00e-4973-bd1e-4bb800aab23f)

**Reply by Gao-pw** [reply]
最开始网上搜索地理可视化的相关方案还都是 echart 和 g2🤡，根本不知道 l7🤯...
